jdi-light/src/main/java/com/epam/jdi/light/logger/JDILogger.java

```diff
@@ -148,6 +180,10 @@ public void info(String s, Object... args) {
     }
     public void error(String s, Object... args) {
         logger.error(jdiMarker, getRecord(s, args));
+        writeToAllure(getRecord(s, args), FAILED);
+        if (screenshotStrategy.equals("on fail")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4558480)